devops.yml

```diff
@@ -0,0 +1,125 @@
+shortlinks:
```

Comment:
This file should go into coala-mobans , as shortlinks.yml

Then in .moban.yml , add `configuration-dir: ../coala-mobans/` like the others.

An immediate use of it in another repo is validating links in the docs .

Reply:
Currently, it's still unavailable.

Reply:
I am still waiting for reviews.  Once it is approved, I will move it and update my PRs and then they can be merged.